scripts/typescript_checks.py

```diff
@@ -260,6 +260,8 @@
     'core/templates/filters/summarize-nonnegative-number.filter.spec.ts',
     'core/templates/filters/summarize-nonnegative-number.filter.ts',
     'core/templates/filters/truncate-input-based-on-interaction-answer-type.filter.ts',
+    'core/templates/filters/truncate-input-based-on-interaction-answer-type.pipe.spec.ts',
```

Comment:
Are we expecting to add new files in this list?

Reply:
@JeeveshGarg is going to solve the type error issue in this file `truncate-input-based-on-interaction-answer-type.pipe`  same error appears in this file too.

Reply:
@DubeySandeep PTAL

Reply:
I see, thanks for the explanation! Do we have any tracking task/issue for the same? (OR maybe you can reply to this thread with the PR link once it's fixed?)
[In any case this comment/thread is not blocking this PR from merging]

Reply:
As jeevesh has to fix this issue, I don't think there is requirement of the tracking issue, I will reply to this thread once the type error got fixed.